src/main/java/com/minecolonies/event/EventHandler.java

```diff
@@ -23,11 +24,26 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import java.util.ArrayList;
+
 /**
  * Handles all forge events.
  */
 public class EventHandler
 {
+
+    /**
+     * Event called when the debug screen is opened.
+     * Event gets cancelled when there no permission to break a hut
+     *
+     * @param event {@link net.minecraftforge.client.event.RenderGameOverlayEvent}
+     */
+    @SubscribeEvent
+    public void onDebugScreenObened(@NotNull RenderGameOverlayEvent.Text event)
+    {
+        System.out.println("workinnn");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS106)